src/Form/FastlySettingsForm.php

```diff
@@ -278,6 +278,13 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#suffix' => '<span class="purge-all-message"></span>',
     ];
 
+    $form['purge']['purge_logging'] = [
```

Comment:
Think we should move this under purge options since this is option for purge.
```suggestion
    $form['purge']['purge_options']['purge_logging'] = [
```